ui_tests/src/test/resources/com/saucedemo/products/signinNegative.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Sign In
+
+  @signIn
+  Scenario: Sign in with invalid password
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'thisIsSomeInvalidPassword' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+
+  @signIn
+  Scenario: Sign in with invalid user name
+    Given I open start page
+    When I set 'thisIsSomeInvalidUser' as username on Sign in page
+    And I set 'standard_password' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign in with empty user and password
+    Given I open start page
+    When I set ' ' as username on Sign in page
+    And I set ' ' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
```

Comment:
Please remove this file. It contains the same tests that you added to the ` ui_tests/src/test/resources/com/saucedemo/account/signinNegative.feature` i.e. I believe it was added to the `products` folder accidentally